components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -45,10 +46,19 @@ class GleanTest {
     @get:Rule
     val fakeDispatchers = FakeDispatchersInTest()
 
+    private fun resetGlean(config: Configuration = Configuration()) {
```

Comment:
This is a nice refactor.  Eventually it probably will need be used by other test modules so maybe belongs in a set of test utilities?  (But that doesn't need to be done for this PR...)

Reply:
This is a good point, but let's wait for the testing API proposal to come up. This might be affected by that API as well.